specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/virtualMachineScaleSet.json

```diff
@@ -1368,6 +1368,60 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/reapply": {
```

Comment:
>reapply [](http://example.com/codeflow?start=141&length=7)

202 must include Location header for long running operations

Reply:
Done.